src/http_server.cpp

```diff
@@ -188,7 +188,7 @@ std::unique_ptr<DrogonHttpServer> createAndStartDrogonHttpServer(const std::stri
 
         std::vector<std::pair<std::string, std::string>> headers;
 
-        for (const std::pair<const std::string, const std::string>& header : req->headers()) {
+        for (const auto& header : req->headers()) {
```

Comment:
We could consider defining type for headers, so it's visible what to expect, but also more convenient than current version. 
But I'm also fine with using `auto` here, so it's up to you.